src/rcc/mod.rs

```diff
@@ -436,13 +437,27 @@ macro_rules! ppre_calculate {
 impl Rcc {
     fn flash_setup(rcc_aclk: u32, vos: Voltage) {
         use crate::stm32::FLASH;
-        let rcc_aclk_mhz = rcc_aclk / 1_000_000;
+        // ACLK in MHz, round down and subtract 1 from integers. eg.
+        // 61_999_999 -> 61MHz
+        // 62_000_000 -> 61MHz
+        // 62_000_001 -> 62MHz
+        let rcc_aclk_mhz = (rcc_aclk - 1) / 1_000_000;
 
         // See RM0433 Table 13. FLASH recommended number of wait
```

Comment:
Nit: RM0433 Rev 7 Table 17

Reply:
Thanks! I think that was written for Rev 6